graphic_templates/waffle_chart/css/graphic.less

```diff
@@ -0,0 +1,23 @@
+@import "base";
+@import "labels";
+
+tspan.value {
+	font-weight: bold;
+}
+
+.labels {
+	font-size: 12px;
+}
+
+.tooltip {
+	background-color: white;
```

Comment:
There are some mixed tabs and spaces in this file—would be nice to keep them consistent as spaces.